### PR TITLE
[skip ci] intro review

### PR DESCRIPTION
**Pull request by Nyco**
* lacked keywords such as "im" or chat" (and "load testing" and "metrics")
* still mentioned Inaka

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,21 +2,22 @@
...
-Designed for enterprise, it is fault-tolerant, can utilise the resources of multiple clustered machines, and easily scales for more capacity by simply adding a box or VM. 
+MongooseIM is a robust and efficient chat (or instant messaging) platform aimed at large installations. 
+Designed for enterprise, it is fault-tolerant, can utilise the resources of multiple clustered machines, and easily scales for more capacity by simply adding a box or VM.
 
-MongooseIM can accept client sessions over vanilla XMPP, Websockets, HTTP long-polling (a.k.a. BOSH), and a REST API.
+MongooseIM can accept client sessions over vanilla XMPP, REST API and SSE, as well as Websockets, and BOSH (HTTP long-polling).
 
-The MongooseIM platform includes server-side components and client libraries. 
-We provide a test suite and a monitoring server. We recommend third-party, open source client libraries for XMPP and REST API.
+The MongooseIM platform includes several server-side (backend and client-side (frontend) components.
```
Comment (fenek) [inline README.md]: Parentheses around "backend" not closed.

File: doc/README.md [hunk 2]
```diff
@@ -2,12 +2,14 @@
...
-Designed for enterprise, it is fault-tolerant, can utilise the resources of multiple clustered machines, and easily scales for more capacity by simply adding a box or VM. 
-MongooseIM can accept client sessions over vanilla XMPP, Websockets, HTTP long-polling (a.k.a. BOSH), and a REST API.
+MongooseIM is a robust and efficient chat (or instant messaging) platform aimed at large installations. 
+Designed for enterprise, it is fault-tolerant, can utilise the resources of multiple clustered machines, and easily scales for more capacity by simply adding a box or VM.
 
-The MongooseIM platform includes server-side components and client libraries. 
-We provide a test suite and a monitoring server. We recommend third-party, open source client libraries for XMPP and REST API.
+MongooseIM can accept client sessions over vanilla XMPP, REST API and SSE, as well as Websockets, and BOSH (HTTP long-polling).
+
+The MongooseIM platform includes several server-side (backend and client-side (frontend) components.
```
Comment (fenek) [inline doc/README.md]: Parentheses not closed.

File: README.md (hunk 1 above)
Comment (goddammit) [inline README.md]: The MongooseIM platform includes several server-side (backend) components and client-side (frontend) tools.

File: doc/README.md (hunk 2 above)
Comment (goddammit) [inline doc/README.md]: The MongooseIM platform includes several server-side (backend) components and client-side (frontend) tools.

**Comment by fenek** [comment]
@goddammit I guess we can stick to "components".

**Comment by Nyco** [comment]
A tool is a stuff to build something else.
A component is used to build on top of it.

Analogy warning:
* a screw is a component of the thing you build, a screwdriver is a tool
* a nail is a component of the thing you build, a hammer is a tool

We don't offer development tools, only components to build on top of.

We address the problem of asset building, helping Make-Or-Buy Decision.

**Pull request merged**